src/bauhaus/bauhaus.h

```diff
@@ -34,6 +34,11 @@
 #include <stdlib.h>
 #include <string.h>
 
+#ifdef __cplusplus
+using namespace std;
```

Comment:
You basically should not ever want to do that,
and you most certainly shouldn't ever do that in public headers.

Reply:
Removed.